doc/source/install/cmake_options.rst

```diff
@@ -0,0 +1,89 @@
+.. -*- rst -*-
+
+CMake options
+=============
+
+This section describes important options of CMake.
+
+To get more details about installing Groonga from the source with CMake on a specific environment,
+find the document for the specific environment from :doc:`/install`.
+
+``-G GENERATOR``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Specify a generator.
+
+This is a mandatory option.
+
+You can check available generators by ``cmake --help``.
+
+Here is an example of specifying ``Visual Studio 12 2013 Win64`` as a generator::
```

Comment:
Visual Studio 2013 reached EOL.
How about using the recent Visual Studio instead?

https://cmake.org/cmake/help/latest/generator/Visual%20Studio%2017%202022.html

(We need to use `-G` and `-A` with newer Visual Studio: https://cmake.org/cmake/help/latest/generator/Visual%20Studio%2017%202022.html#platform-selection )

Reply:
Fixed.